libp2p/Host.h

```diff
@@ -317,7 +317,9 @@ class Host: public Worker
 	
 	/// Peers we try to connect regardless of p2p network.
 	std::set<NodeID> m_requiredPeers;
-	Mutex x_requiredPeers;
+	Mutex mutable x_requiredPeers;
+	/// returns true if a member of m_requiredPeers
+	bool isRequiredPeer(NodeID const&) const;
```

Comment:
I'd prefer not to intermix data member declarations with method declarations (so I'd move this line above all data members)

Also `mutable` is usually before the type name.

Reply:
OK.